src/commands/lightning/preview/app.ts

```diff
@@ -202,46 +204,68 @@ export default class LightningPreviewApp extends SfCommand<void> {
       targetOrg = this.argv[idx + 1];
     }
 
+    // Start the LWC Dev Server
+    const lwcServer = await startLWCServer(sfdxProjectRootPath, logger);
+    const serverPort = lwcServer.context?.config?.port ?? LOCAL_DEV_SERVER_DEFAULT_PORT;
+
+    logger.debug('Determining Local Dev Server url');
+    const ldpServerUrl = PreviewUtils.generateWebSocketUrlForLocalDevServer(Platform.desktop, serverPort);
+    logger.debug(`Local Dev Server url is ${ldpServerUrl}`);
+
     const launchArguments = PreviewUtils.generateDesktopPreviewLaunchArguments(ldpServerUrl, appId, targetOrg);
 
-    // Start the LWC Dev Server
-    await startLWCServer(process.cwd(), logger ? logger : await Logger.child(this.ctor.name));
+    // todo: For the desktop scenario, should we print a message to the terminal screen to inform the user
```

Comment:
- We probably should.
- We may want to wait until we have a link to documentation that could further describe it. But if you wanted to do a placeholder in this PR, you could just keep it terse:

> Note: Your desktop browser requires additional configuration to trust the local development server. See the docs for more details.